tests/integration/annotation_import/conftest.py

```diff
@@ -160,19 +160,19 @@ def configured_project_pdf(client, ontology, rand_gen, pdf_url):
 
 @pytest.fixture
 def configured_project_without_data_rows(client, configured_project, rand_gen):
-    project = client.create_project(name=rand_gen(str))
+    project = client.create_project(name=rand_gen(str),
```

Comment:
Could we also add this extra param in the example docstring here?

https://github.com/Labelbox/labelbox-python/blob/3b375d49b5e9ade23151e717876a4e4f3f312881/labelbox/client.py#L614